src/run.jl

```diff
@@ -43,11 +62,107 @@ julia> sol.t[end]
 0.1
 ```
 """
-function trixi_include(mod::Module, parameters_file::AbstractString; kwargs...)
-  Base.include(ex -> replace_assignments(ex; kwargs...), mod, parameters_file)
+function trixi_include(mod::Module, elixir::AbstractString; kwargs...)
+  Base.include(ex -> replace_assignments(ex; kwargs...), mod, elixir)
 end
 
-trixi_include(parameters_file::AbstractString; kwargs...) = trixi_include(Main, parameters_file; kwargs...)
+trixi_include(elixir::AbstractString; kwargs...) = trixi_include(Main, elixir; kwargs...)
+
+
+"""
+    convergence_test(elixir::AbstractString, iterations; kwargs...)
+
+Run multiple Trixi simulations with the parameters given in `elixir` and compute
+the experimental order of convergence (EOC) in the ``L^2`` and ``L^\\infty`` norm.
+The number of runs is specified by `iterations`; in each run the initial
+refinement level will be increased by 1. Parameters can be overriden by specifying them as
+additional keyword arguments, which are passed to the respective call to `trixi_include`.
+"""
+function convergence_test(elixir::AbstractString, iterations; kwargs...)
+  @assert(iterations > 1, "Number of iterations must be bigger than 1 for a convergence analysis")
+
+  # Types of errors to be calcuated
+  errors = Dict(:L2 => Float64[], :Linf => Float64[])
```

Comment:
Shouldn't we use `l2` and `linf` here as we do in the return value from `analysis_callback` for, you know, consistency? Even if L2/Linf was used before, I don't see a strong reason to keep the names

Reply:
Done in 99539f9.